.editorconfig

```
@@ -230,3 +230,231 @@ dotnet_naming_style.I_style.capitalization = pascal_case
 # RS1010: Provide an explicit argument for optional parameter 'equivalenceKey', which is non-null and unique across all code actions created by this fixer.
 dotnet_diagnostic.RS1010.severity = none
 
+# ----------------------------------------------------------------------------------------------------------------------
+# SyleCop.Analyzers rules - note that the URLs below are for tag 1.1.118
+# ----------------------------------------------------------------------------------------------------------------------
+
+# Spacing Rules (SA1000-) https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/1.1.118/documentation/SpacingRules.md
+
+dotnet_diagnostic.SA1000.severity = warning
+dotnet_diagnostic.SA1001.severity = warning
+dotnet_diagnostic.SA1002.severity = warning
+dotnet_diagnostic.SA1003.severity = warning
+dotnet_diagnostic.SA1004.severity = warning
+dotnet_diagnostic.SA1005.severity = warning
+dotnet_diagnostic.SA1006.severity = warning
+dotnet_diagnostic.SA1007.severity = warning
+dotnet_diagnostic.SA1008.severity = warning
+dotnet_diagnostic.SA1009.severity = none # needs discussion
+dotnet_diagnostic.SA1010.severity = warning
+dotnet_diagnostic.SA1011.severity = warning
+dotnet_diagnostic.SA1012.severity = none # noisy on collection initializers
+dotnet_diagnostic.SA1013.severity = none
```

I would repeat the comment, as it's not obvious from the rule ID that these two are related.
```suggestion
dotnet_diagnostic.SA1013.severity = none # noisy on collection initializers
```